.github/workflows/maven.yml

```diff
@@ -59,15 +59,15 @@ jobs:
 
       - name: Build with Maven
         run: ./mvnw -e -B -V -ntp clean install
-        
-      - name: Build site
-        run: ./mvnw -e -B -V -ntp site:site
 
       - name: Generate javadoc
-        run: ./mvnw -e -B -V -ntp javadoc:javadoc 
+        run: ./mvnw -e -B -V -ntp javadoc:javadoc
```

Comment:
In the past we had javadoc build failures that strangely did not break the site build and were only discovered in the release-preparation phase. Thus this build step is a "security net" to make sure that javadoc can be properly generated for the whole project!